doc/source/report/ultradian.rst

```diff
@@ -1,92 +1,66 @@
-  tool in period detection and frequency analysis.
-
-What is the connection with other subprojects?
-
-- Ultradian rhythms could be treated as one feature for clustering the 16
-  strains. We may also subset the data using the results of the cluster and
-  analysis the rhythm similarities and differences across clusters.
+Ultradian and circadian rhythm are widely observed in mammalian
+behavioral patterns. The ultradian analysis aims to
+find the time-specific patterns in behavioral
+records, without specifying the length of the cycle in advance (but need to be
+within 1 hour to 1 day). The typical ultradian period includes 8 and 12 hours.
+The circadian rhythm refers to a roughly 24-hour cycle.
+For example, we expect the rats to be inactive in the nighttime.
+And ingestions and movements mostly happened in the daytime.
+
+The variables of interests are the summary statistics of mouse activity
+including food (F) and water (W) ingestion, distance traveled (D), Active
+State probability (AS), movement inside the home base (M_IS) and
+movement outside the home base (M_AS). We may also consider spatial variables,
+for example, to spatially discrete the data to cells each with its primary
+functions such as food cell, water cell, and examine
+ultradian cycle of the spatial probability densities
+of the occupancy time in each cell.
+
+In this report, we assume strain to be the primary influence on the variation of
+ultradian rhythms across individual mouse and examine the
+difference in rhythms across strains. Currently, we lack
+basic information such as weight, age, and gender. With more data available
+in the future, we may look into the cycle for each mouse and detect the most
+important factors influencing the ultradian rhythms.
+
+The ultradian and circadian analysis is closely related to other subprojects.
+Ultradian rhythms could be treated as one feature for clustering the 16
+strains. We may also subset the data using the results of the cluster and
+analysis the rhythm similarities and differences across clusters.
 
 Statement of statistical problem
 --------------------------------
 
-How to determine the optimal bin intervals for constructing the time series?
-
--  The bin interval may vary according to the frequency. Bin interval
-   examples: 5 min, 30 min, 1 hour etc. Need to look into the data.
-
-How to test the autocorrelation coefficient for the data and assess the model?
+Our statistical problems are three parts: data preparation, choice of
+the frequency or period, and modeling of rhythm patterns.
 
--  Use the AIC to select best time lags for the time series model and
-   the K statistics to test the goodness of fit.
+- Data preparation: mouse behaviour are recorded based on time intervals
+  or time points. For example, the beginning and ending time
+  stamp of one food ingestion, or the coordinates of mouse
+  position at a specific time point. We aggregate the
```

Comment:
"or time points, like the beginning and ending time
stamp of one food ingestion or the coordinates of mouse
position at a specific time point."